charts/kubermatic/static/master/provider-incompatibilities.yaml

```diff
@@ -0,0 +1,13 @@
+# This file has been generated using hack/update-kubermatic-chart.sh, do not edit.
```

Comment:
I appreciate the effort, but we do not update this chart anymore. It's soon going to be deleted entirely, so you can skip everything needed to generate this file.

Reply:
NB: We _still_ required the seed-ctrlmgr to read data from files (instead of the `KubermaticConfiguration`), but work is being done to change that (i.e. we do not want to have a versions.yaml, updates.yaml etc. anymore, all we want is one single KubermaticConfiguration CRD and that's it).

So we might still need code to generate this file, but we do not update this Helm chart anymore. The KKP operator must create the file instead.

Reply:
@xrstf I had to recreate the file since it is used (along `with versions.yaml` and `updates.yaml`) by the `run-api.sh` script. If we don't statically provide this file to it, that test script is unusable.